Updated language in docs for global UI project (#6461)
fishtown-analytics/docs.getdbt.com: website/docs/guides/azure-synapse-analytics-qs.md

## Connect dbt Cloud to Azure Synapse Analytics

> 1. Create a new project in dbt Cloud. Navigate to your account name in the left side menu, select **Account settings**, and click **+ New Project**.

Reviewer: 🔥

> 1. Create a new project in dbt Cloud. Navigate to your account name in the left side menu, select **Account settings**, and click **+ New Project**.

Reviewer: ```suggestion
1. Create a new project in dbt Cloud. Click on your account name in the left side menu, select **Account settings**, and click **+ New Project**.
```

Reviewer: i think we should specify we 'click' on the account name, not navigate to it
